bin/travis-build.bash

```diff
@@ -3,56 +3,84 @@ set -e
 
 echo "This is travis-build.bash..."
 
+echo "-----------------------------------------------------------------"
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
-sudo apt-get install postgresql-$PGVERSION solr-jetty libcommons-fileupload-java:amd64=1.2.2-1
+sudo apt-get install solr-jetty libcommons-fileupload-java libpq-dev postgresql postgresql-contrib
 
+pip install setuptools -U
+
+# if [ $CKANVERSION == '2.8' ]
+# then
+# 	sudo apt-get install postgresql-9.6
+# elif [ $CKANVERSION == '2.3' ]
+# then
+# 	sudo apt-get install postgresql-9.1
+# fi
+
+echo "-----------------------------------------------------------------"
 echo "Installing CKAN and its Python dependencies..."
-git clone https://github.com/gsa/ckan
+
+cd .. # CircleCI starts inside ckanext-datajson folder
+pwd
+ls -la
+
+git clone https://github.com/ckan/ckan
 cd ckan
-if [ $CKANVERSION == 'inventory' ]
```

Comment:
Because Data.gov has some customization on our [ckan fork](https://github.com/GSA/ckan), we probably want to include these branches in our tests:
- [inventory](https://github.com/GSA/ckan/tree/inventory)
- [datagov](https://github.com/GSA/ckan/tree/datagov)

Reply:
Added this enverinment also
https://github.com/datopian/ckanext-datajson/commit/29fe7479576f17ab0876c180bed8fefd74603d30